README.md

```diff
@@ -54,6 +54,11 @@ $ ./script/build
 $ cp hub YOUR_BIN_PATH
 ~~~
 
+# Or, if your go development bin directory is already in your $PATH:
```

Comment:
Ah, gotcha; so it's `$GOPATH/bin`. Maybe we should say that here to avoid ambiguity?